README.md

```diff
@@ -225,6 +225,7 @@ const authClient = new auth.OAuth2User({
 
 const authUrl = authClient.generateAuthURL({
   code_challenge_method: "S256",
+  authorizeUrl: "https://getalby.com/oauth" // endpoint for authorization (replace with the appropriate URL based on the environment)
```

Comment:
if it's put in the example like this, users will think they need to pass it. Should it be commented out here?

Reply:
resolved